slack_etiquette.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: Slack Etiquette
+permalink: /about/slacketiquette
+published: true
+---
+
+
+# Slack Etiquette
+### Aka meming and messaging responsibly - msoe-sse.slack.com
+> Few people want constant phone buzzes 
+
+## 1. Unless you are on the executive board, do NOT use @channel in #general or #random under any circumstances
+> If you do have something of importance, please message _Brady Cassada_, the Communications Coordinator
+
+## 2. Threads are your friend! They don't clog up the channel, and they keep conversation focused. 
+> In your **thread reply**, be sure to **@whoever** you're replying to in the thread otherwise they might not get a notification!

```

Comment:
Maybe include a screenshot of a thread here? For a newcomers, the idea of threads in Slack can be a little confusing (this is coming from my 35-year old coworker, so I promise I'm not making this up!).